udata/templates/reuse/display.html

```diff
@@ -24,35 +24,6 @@
     <li class="active">{{ reuse.title }}</li>
 {% endblock %}
 
-{% block toolbar %}
```

Comment:
It's not used anymore (legacy from dark ages long time ago ;) ).
I'll submit another PR to remove it in other templates.